### ISSUE TITLE
A11y: CTooltip not dismissed on `esc` for focused Anchors

### ISSUE THREAD

**Issue by codebender828** [chakra-ui/chakra-ui-vue#118]
**Describe the bug**
According to WAI-ARIA specifications for tooltips, the tooltip should be be dismissed if the anchor is focused on `Escape` keyboard event. Currently if the tooltip anchor is a focusable element (e.g. a Button) tooltip will correctly show on focus, but is not dismissed when the `Escape` key is pressed.

See: https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-21

**To Reproduce**
1. Go to 'https://chakra-ui-vue.netlify.com/?path=/story/ui-tooltip--with-custom-separator'
2. Tab until button is focused. Tooltip should be shown
3. Press Escape.
4. Tooltip remains visible.
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/21237954/78782509-bbdbe700-79d4-11ea-9065-6987f9f081df.gif)

**Expected behavior**
Tooltip should be dismissed on escape for focused tooltip anchor
Example: https://cdpn.io/Moiety/debug/LaPvWy
![ezgif com-video-to-gif (2)](https://user-images.githubusercontent.com/21237954/78782519-bf6f6e00-79d4-11ea-8e5c-bcf72e3e1b82.gif)
